core/tests/test_utils.py

```diff
@@ -3030,12 +3033,35 @@ def get_response_without_checking_for_errors(
                 params, dict,
                 msg='Expected params to be a dict, received %s' % params)
 
+            if http_method != 'GET':
+                self.assertIn(
+                    http_method, ['POST', 'PUT', 'DELETE'])
+
         # This swap is required to ensure that the templates are fetched from
         # source directory instead of webpack_bundles since webpack_bundles is
         # only produced after webpack compilation which is not performed during
         # backend tests.
         with self.swap(base, 'load_template', mock_load_template):
-            response = self.testapp.get(url, params=params, expect_errors=True)
+
+            if http_method == 'GET':
+                response = self.testapp.get(
+                    url, params=params, expect_errors=True
+                )
+
+        if http_method == 'POST':
+            response = self.testapp.post(
+                url, params=params, expect_errors=True
+            )
+
```

Comment:
Drop this newline and the one below since all of these are part of the same if/elif/else block.

Reply:
Done.